app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> list:
+        total_wash_price = 0
+        for car in cars:
+            if self.wash_single_car(car):
+                total_wash_price += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+
+        return total_wash_price
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            (car.comfort_class * (self.clean_power - car.clean_mark)
+             * self.average_rating) / self.distance_from_city_center,
+            1)
```

Comment:
Code Style: This is a long mathematical operation. It could be split into additional variables or broken after binary operations to improve readability. Refer to the checklist for good and bad examples.